src/global_legacy_module/4C_global_legacy_module_validmaterials.cpp

```diff
@@ -1585,6 +1586,14 @@ std::shared_ptr<std::vector<std::shared_ptr<Mat::MaterialDefinition>>> Global::v
     m->add_component(
         entry<double>("INITVOL", {.description = "Initial volume of RVE", .default_value = 0.0}));
 
+    m->add_component(selection<int>("RUNTIMEOUTPUT_GP",
```

Comment:
```suggestion
    m->add_component(selection<Mat::PAR::MicroMaterial>("RUNTIMEOUTPUT_GP",
```

The new input mechanism allows to store the enum directly. Most cases we have in the code are not using that because this was not possible before.